python/lsst/ts/m2/mock_command.py

```diff
@@ -139,58 +264,105 @@ async def reset_force_offsets(self, message, message_event):
         ----------
         message : `dict`
             Command message.
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
         message_event : `MockMessageEvent`
             Instance of MockMessageEvent to write the event.
+
+        Returns
+        -------
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
+        `CommandStatus`
+            Status of command execution.
         """
 
-        # TODO (DM-30851):
-        # Implement in DM-30851
-        pass
+        model.reset_force_offsets()
+
+        return model, CommandStatus.Success
 
-    async def clear_errors(self, message, message_event):
+    async def clear_errors(self, message, model, message_event):
         """Clear the system errors.
 
         Parameters
         ----------
         message : `dict`
             Command message.
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
         message_event : `MockMessageEvent`
             Instance of MockMessageEvent to write the event.
+
+        Returns
+        -------
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
+        `CommandStatus`
+            Status of command execution.
         """
 
-        # TODO (DM-30851):
-        # Implement in DM-30851
-        pass
+        model.clear_errors()
+
+        return model, CommandStatus.Success
 
-    async def switch_force_balance_system(self, message, message_event):
+    async def switch_force_balance_system(self, message, model, message_event):
         """Switch the force balance system.
 
         Parameters
         ----------
         message : `dict`
             Command message.
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
         message_event : `MockMessageEvent`
             Instance of MockMessageEvent to write the event.
+
+        Returns
+        -------
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
+        `CommandStatus`
+            Status of command execution.
         """
 
-        await message_event.write_force_balance_system_status(message["status"])
+        command_success = model.switch_force_balance_system(message["status"])
+
+        await message_event.write_force_balance_system_status(
+            model.force_balance_system_status
+        )
+
+        return model, CommandStatus.Success if command_success else CommandStatus.Fail
```

Comment:
consider doing `command_success == True`

Reply:
Done in the commit: e99800d. Thanks!